Bonsai.Editor/EditorForm.Designer.cs

```diff
@@ -1317,7 +1316,6 @@ private void InitializeComponent()
             // 
             // workflowFileWatcher
             // 
-            this.workflowFileWatcher.EnableRaisingEvents = false;
```

Comment:
For posterity: This is the default value for `EnableRaisingEvents`, this is just the designer cleaning up a useless assignment.